util/projectRequest.ts

```diff
@@ -3,6 +3,43 @@ import { paginateRest } from '@octokit/plugin-paginate-rest';
 import githubColorsFixture from '../data/githubColors.json';
 import { Project, ACMCommitteeTopics, GitHubColors } from './types';
 
+export async function getUclaOpenSource(): Promise<Project[]> {
+  const PaginatedOctokit = Octokit.plugin(paginateRest);
+  const octokit = new PaginatedOctokit();
```

Comment:
Similar comment to [this pr!](https://github.com/uclaacm/opensource/pull/97/files#r917318848)
```suggestion
  const octokit = new Octokit.plugin(paginateRest);
```

instead of 
```
  const PaginatedOctokit = Octokit.plugin(paginateRest);
  const octokit = new PaginatedOctokit();
```
#